fix: mention no space around case

OnkarRuikar opened the pull request:
- fixes https://github.com/mdn/content/issues/28460

The PR fixes the
> The CSS Standards mix examples using spaces and omitting spaces, for clarity.

...part

estelle commented:
I disagree with these edits. We should show best practices, which is legible code (with spaces). We can add a comment, similar to how the specs add a single occurrence with a comment, 

`The white space around the `>` combinator is optional but recommended.`

OnkarRuikar committed:
fix: mention optional whitespace around combinators is recommented

OnkarRuikar commented:
> We can add a comment, similar to how the specs add a single occurrence with a comment,

I've added the comments to the syntax sections and removed the without space use.

estelle reviewed (approved):
Thanks! 🎉
